.github/scripts/run_consensus.sh

```diff
@@ -7,11 +7,8 @@ mkdir -p docker/data/
 # source account ids
 source docker/env
 
-# Generate chainspec and populate comittee keystores
-# hidden assumption: --chain-id must be present and different from a0dnet1, otherwise default chainspec
```

Comment:
for now, I would say that after removing the flag, we start expecting `chain-id` set in other way -- what do I miss? how is it set now?

Reply:
`--chain-id` has a default value `a0dnet1`
https://github.com/Cardinal-Cryptography/aleph-node/blob/main/bin/node/src/chain_spec.rs#L112

we use that assumption here now and in `run_nodes.sh`, ie we don't pass `--chain-id` explictely. This was a technical debt introduced in the very first work on staking, which this PR tries to address finally.